src/api/user/info/useInfoFetch.ts

```
@@ -11,5 +11,9 @@ interface UserInfoFetchResponse extends ResponseModel {
 }
 
 export const userInfoFetch = async () => {
-  return await withAuthInstance.get<UserInfoFetchResponse>('/user/me/info');
+  try {
+    return await withAuthInstance.get<UserInfoFetchResponse>('/user/me/info');
+  } catch (e) {
+    throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)